cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/replication/PullFilter.java

```diff
@@ -96,9 +96,11 @@ public PullFilter(String filterName) {
      */
```

Comment:
javadoc is wrong (and was probably wrong before):
- `"Constructs a filter object for a function that requires no parameters"`: obviously not true
- `@param parameters Any parameters required for the function. Can be {@code null}`: can't be null as we will NPE
- `Integer values should be added as String objects`: this is no longer true since we don't use a query string. I wonder if it's a breaking change?